cli/src/alluxio.org/cli/cmd/info/collect.go

```diff
@@ -0,0 +1,147 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package info
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/palantir/stacktrace"
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/env"
+)
+
+var Collect = &CollectCommand{
+	BaseJavaCommand: &env.BaseJavaCommand{
+		CommandName:   "collect",
+		JavaClassName: "alluxio.cli.bundler.CollectInfo",
+	},
+}
+
+type CollectCommand struct {
+	*env.BaseJavaCommand
+
+	additionalLogs       []string
+	endTime              string
+	excludeLogs          []string
+	excludeWorkerMetrics bool
+	includeLogs          []string
+	local                bool
+	maxThreads           int
+	startTime            string
+}
+
+func (c *CollectCommand) Base() *env.BaseJavaCommand {
+	return c.BaseJavaCommand
+}
+
+const dateFormat = "2006-01-02T15:04:05"
+
+func (c *CollectCommand) ToCommand() *cobra.Command {
+	cmd := c.Base().InitRunJavaClassCmd(&cobra.Command{
+		Use:   fmt.Sprintf("%v [command] [outputPath]", Collect.CommandName),
```

Comment:
The current command currently looks like `bin/cli.sh info collect {all | collectAlluxioInfo. | collectConfig | collectEnv | collectJvmInfo | collectLog | collectMetrics} /path/to/output`. Here are some suggestions for improvements:
Trim commands to become `bin/cli.sh info collect {all | cluster | conf | env | jvm | log | metrics }`. Turn the output path into a flag like `-o, --output-path`, make the default value `pwd` or make flag mandatory.